problemsets-mat223/linearalbegra.tex

```diff
@@ -4743,13 +4743,38 @@ \section*{Inverses}
 
 \subsection*{More Change of Basis}
 	\question
-	Let $\mathcal B=\{\vec b_1,\vec b_2\}$ where $\vec b_1=\mat{1\\1}$, $\vec b_2=\mat{1\\-1}$
+	Let $\mathcal B=\Set{\vec b_1,\vec b_2}$ where $\vec b_1=\mat{1\\1}$, $\vec b_2=\mat{1\\-1}$
 	and let $X=[\vec b_1|\vec b_2]$ be the matrix whose columns are $\vec b_1$ and $\vec b_2$.
 	\begin{parts}
 		\item Compute $[\xhat]_{\mathcal B}$ and $[\yhat]_{\mathcal B}$.
+			\begin{solution}
+				$[\xhat]_{\mathcal B}=\frac{1}{2}\mat{1\\1}$ and
+				$[\yhat]_{\mathcal B}=\frac{1}{2}\mat{1\\-1}$.
+
+				This is because $\xhat=\frac{1}{2}(\vec b_1+\vec b_2)$ and
+				$\yhat=\frac{1}{2}(\vec b_1-\vec b_2)$
+			\end{solution}
 		\item Compute $X[\xhat]_{\mathcal B}$ and $X[\yhat]_{\mathcal B}$.  
 			What do you notice?
+			\begin{solution}
+				$X[\xhat]_{\mathcal B}=\frac{1}{2}\mat{1&1\\1&-1}\mat{1\\1}=\mat{1\\0}$
```

Comment:
Add the intermediate step of `1/2\mat{1\\1} + 1/2\mat{1\\-1}=1/2b_1+1/2b_2=e_1` and then we don't need the `seems` in the next paragraph, because it's obvious.

Reply:
Sorry, I don't think I'm clear on what you mean here. The line you've highlighted is just a computation of matrix acting on a vector, and I'm not sure how a sum of two vectors would be an intermediate step. 

I'm happy to just delete the `seems`, since it's already pretty obvious I guess.

Reply:
What I'm trying to point out is that you should refer to the vectors `b_1` and `b_2` during your computation.